standard-app/templates/configs/configmap.yaml

```diff
@@ -0,0 +1,42 @@
+# Default configMap
+{{- if .Values.configMap }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-configmap
+data:
+{{- range $key, $value := .Values.configMap }}
+  {{ $key }}: {{ $value }}
+{{- end }}
+---
+{{- end }}
+
+# Deployment configMap
+{{- if .Values.apps.configMap }}
```

Comment:
This would create a single configmap for all deployments.
Let's use `range` to iterate through all `apps` and create a configmap per app where `configMap` key is specified (similar to deployment level secrets)

Reply:
Shouldn't we iterate over jobs and cronjobs as well?

Reply:
`range` was used to iterate apps, cronjobs, and jobs, to create configMap